c2corg_ui/static/js/auth/auth.js

```diff
@@ -151,10 +161,19 @@ app.AuthController.prototype.successRegister_ = function(response) {
  * @private
  */
 app.AuthController.prototype.errorRegister_ = function(response) {
-  // TODO
-  alert('register error');
-  console.log('register error');
-  console.log(response);
+  // TODO: i18n
+  // FIXME: HTML is not interpreted in alert messages
+  var msg = 'Registration failed because of:';
+  var errors = response['data']['errors'],
+      len = errors.length;
+  if (len > 0) {
+    msg += '<ul>';
```

Comment:
> But then the "close" button is lost :(

The problem is the combination of the ``ng-include`` in AngularUI Bootstrap alert (used to pass the content to display to the alert template) and of ``trustAsHtml`` filter since the latter requires to change
```
<uib-alert ng-repeat="alert in alertCtrl.alerts" template-url="/static/partials/alert-tpl.html">{{alert.msg}}</uib-alert>
```
to
```
<uib-alert ng-repeat="alert in alertCtrl.alerts" template-url="/static/partials/alert-tpl.html" ng-bind-html="{{alert.msg | appTrustAsHtml}}"></uib-alert>
```
"ng-bind-html" then replaces most of the content of ``/static/partials/alert-tpl.html`` including the "close" button code.

We could also skip using AngularUI Bootstrap alert, using instead a single template with no "ng-include" but with the "appTrustAsHtml" filter. It works: the formatted message as well as the close button are displayed. But then we lose the timeout closing feature, which requires a controller instance for each alert message (and its own $timeout)...
See https://github.com/angular-ui/bootstrap/blob/master/src/alert/alert.js